src/Orleans/CodeGeneration/TypeUtils.cs

```diff
@@ -561,7 +561,7 @@ public static IEnumerable<TypeInfo> GetDefinedTypes(Assembly assembly, Logger lo
             }
             catch (Exception exception)
```

Comment:
Yeah, I agree with Reuben, there's no reason to have a breaking change with this behavior now. Especially since we'll revamp assembly loading soon.